modules/datastore/src/Form/ResourceSettingsForm.php

```diff
@@ -54,6 +54,16 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $this->config('datastore.settings')->get('delete_local_resource'),
       '#description' => $this->t('Delete local copy of remote files after the datastore import is complete'),
     ];
+    $form['drop_datastore_on_post_import_error'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Drop the datastore table if the post import queue reports an error.'),
+      '#default_value' => $this->config('datastore.settings')->get('drop_datastore_on_post_import_error'),
+      '#description' => $this->t('The datastore import queue brings in all columns as strings. The post import
+      queue will alter the table according to the data dictionary, if there is a problem during this step the
+      error will be posted to the Datastore Import Status dashboard, and the datastore table will keep all
+      data typed as strings. Check this box if you prefer that the table be dropped if there is a problem
+      in the post import stage.'),
```

Comment:
Just a thought here while it could be redundant (and as someone who is not familiar at all with this process), would it make sense and be accurate here to have this last sentence reconfirm what would happen otherwise, like: "Check this box if you prefer that the table be dropped if there is a problem, rather than have it keep all data typed as strings."

Reply:
The config is not concerned with the type of error at this time, it could be that there is no data typing happening so strings would be fine, but there was some other issue, like a parsing error, file encoding...